source/http_connection.c

```diff
@@ -135,7 +135,13 @@ static void s_on_client_connection_setup(
     }
 
     /* Invoke on_setup, then clear our reference to it */
-    PyObject *result = PyObject_CallFunction(connection->on_setup, "(Oi)", capsule ? capsule : Py_None, error_code);
+    PyObject *result = PyObject_CallFunction(
+        connection->on_setup,
+        "(Oii)",
+        capsule ? capsule : Py_None,
+        error_code,
+        aws_http_connection_get_version(native_connection));
```

Comment:
`native_connection` will be NULL if the `error_code` was non-zero.
probably want to start with 
`enum aws_http_connection_version http_version = AWS_HTTP_VERSION_UNKNOWN;`
and set it for real in the `if (!error_code)` branch above